app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="org.mozilla.focus">
+
+    <!--For fastlane usage-->
+    <!-- Allows changing locales -->
+    <uses-permission
+        android:name="android.permission.CHANGE_CONFIGURATION"
```

Comment:
I'm a little concern about in the future, the developer will have this permission and break some features without notice. I can think of two work around
1. have another build type for screengrab
2. manually paste or programmatically generate this file when we want to run screengrab jobs.

Previously we had a bug that we want to run UI tests on private mode activity so we change its process to be the same as the testing app. And the developers didn't aware of that in dev build so had a strange regression.